src/types/mod.rs

```diff
@@ -177,6 +177,24 @@ impl Flag<'static> {
     }
 }
 
+impl<'flag> Flag<'flag> {
+    /// Converts a `Flag<'flag>` into a `Flag<'static>` or in other words: It's
+    /// making sure that `Flag::Custom` includes a `String` and not a `&str`
+    /// anymore.
+    pub fn into_static(&self) -> Flag<'static> {
```

Comment:
We've generally been calling these methods [`into_owned`](https://docs.rs/imap/3.0.0-alpha.4/imap/types/enum.AttributeValue.html#method.into_owned), but otherwise very much on board with this change!